tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile

```diff
@@ -181,14 +181,14 @@ pipeline {
         stage("Build and push 'che-dashboard' image"){
             steps {
                 sh """
-                    echo "Download the 'che-dashboard-next' project and checkout to branch"
```

Comment:
"che-dashboard" is common name now, isn't it?

Reply:
Good catch! Find/replace mistake.

Removed dashboard next notion here and in other places https://github.com/eclipse/che/pull/19618/commits/cf4fd2f896a1a0413ba802e67e7e363e79f58fdf